cockroachdb/templates/serviceaccount-certRotateSelfSigner.yaml

```diff
@@ -13,4 +13,10 @@ metadata:
   {{- with .Values.labels }}
     {{- toYaml . | nindent 4 }}
   {{- end }}
+  {{- if .Values.tls.serviceAccount.annotations }}
+  annotations:
+  {{- with .Values.tls.serviceAccount.annotations }}
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- end }}
```

Comment:
creating a separate service account annotations config in `values.yaml` for self signer jobs instead of using `.Values.tls.serviceAccount.annotations`